py3status/modules/wifi.py

```diff
@@ -11,9 +11,8 @@
     device: Wireless device name (default "wlan0")
     down_color: Output color when disconnected, possible values:
         "good", "degraded", "bad" (default "bad")
-    format_down: Output when disconnected (default "W: down")
-    format_up: See placeholders below
-        (default "W: {bitrate} {signal_percent} {ssid}")
+    format: Display format for this module
+        (default 'W: [{bitrate} {signal_percent} {ssid}|\?show down]')
```

Comment:
I see for some reason you have `if not bool(self.py3.get_placeholders_list(self.format)):` which is pointless.

you need to check if the format has been changed by the user so `if self.format == DEFAULT_FORMAT:` of course you will need to specify `DEFAULT_FORMAT` etc